### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by QKaiser**

### CODE REVIEW

**Comment by kissgyorgy** [comment]
I think Discussions should be used some categories, as it has already topics for questions and suggestions, and has better features for threaded replies and long-form discussions.

**Comment by QKaiser** [comment]
> I think Discussions should be used some categories, as it has already topics for questions and suggestions, and has better features for threaded replies and long-form discussions.

Which ones do you suggest we use Discussions for ?

**Comment by QKaiser** [comment]
> I would keep bug report for sure, maybe file format suggestion, with keeping in mind that it's perfectly fine to close a request :) and just redirect all other questions, ideas to discussions. I have seen workflows in other projects where they require a discussion first, only after it has been discussed they allow creating issues, so the issue tracker only reflects work they actually want to do. We don't have to be so strict, but I think that's a good idea.

Got rid of questions, we can use Discussions for that. Kept bug report, format request, and feature request.

File: .github/ISSUE_TEMPLATE/bug-report-🐞.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment information (please complete the following information):**
+ - OS: [e.g. Ubuntu Linux]
+ - Software versions (Docker, Python, Nix, Poetry, if applicable)
```
[the remark this answers is not in the document]
Comment (QKaiser) [reply .github/ISSUE_TEMPLATE/bug-report-🐞.md:47]: Fixed. Also provided commands to get Linux/MacOS distribution info and Linux kernel version.

File: .github/ISSUE_TEMPLATE/bug-report-🐞.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
[the remark this answers is not in the document]
Comment (QKaiser) [reply .github/ISSUE_TEMPLATE/bug-report-🐞.md]: Fixed.

**Review by vlaci — APPROVED** [review]

**Pull request merged**